buildscripts/travis-build.sh

```diff
@@ -14,10 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+MASTER="master"
```

Comment:
Can we have unit test script & linting against this .sh file.

Reply:
ref - https://medium.com/@pimterry/testing-your-shell-scripts-with-bats-abfca9bdc5b9

Reply:
ref  - https://www.shellcheck.net/

Reply:
as discussed,  added how the program is tested locally.

Reply:
@gkGaneshR where has this been added

Reply:
on commit message